api/src/main/java/org/pmiops/workbench/db/dao/DataSetService.java

```diff
@@ -40,4 +42,8 @@ DataSet cloneDataSetToWorkspace(
       DataSet fromDataSet, Workspace toWorkspace, Set<Long> cohortIds, Set<Long> conceptSetIds);
 
   List<DataSet> getDataSets(Workspace workspace);
+
+  List<ConceptSet> getConceptSets(DataSet dataSet);
```

Comment:
Why is this in `DataSetService` instead of the `ConceptSetService`?

If must be in `DataSetService`, why is it not called `getConceptSetsForDataSet`?

Reply:
Even in `ConceptSetService` I would consider calling it `getConceptSetsForDataSet`, come to think of it, and the other gets `getConceptSetsForWhateverTheArgumentIs`

Reply:
My reasoning was that the `DataSet` model is the one that maintains the relationship to the `ConceptSet` and `Cohort` models so its service should be responsible for performing the fetch.

Neither `ConceptSet` or `Cohort` has any reference to `Dataset` so I view them as a base level of functionality on top of which `Dataset` has been built.

I opted for the shorter function names because I felt like the function signature captures the `ForDataset` context.

Ex. For me, I gain the same context from reading `getConceptSets(dataSet)` as I do from `getConceptSetsForDataset(dataSet)`.

I am cool with changing them to the longer version though.

Reply:
Thanks for the explanation, that makes sense.